### PR TITLE
Config for jenkins saucelab testing

### PR DESCRIPTION
**Pull request by dpford**
This is the config file needed to get saucelabs working in Jenkins

How to test: Run the functional testing jenkins job

Review: @jimmynotjim

### CODE REVIEW

File: _tests/browser_tests/jenkins_conf.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    'browserName': 'chrome',
+    'name': 'flapjack-browser-tests'
+  },
+  sauceUser: process.env.SAUCE_USER,
+  sauceKey: process.env.SAUCE_KEY,
+
+  onPrepare: function() {
+    return browser.ignoreSynchronization = true;
+  }
+};
```
Comment (jimmynotjim) [inline _tests/browser_tests/jenkins_conf.js]: Add an EOF newline

**Comment by anselmbradford** [comment]
Probably easiest to close this PR And re-open a new one to get the history correct.

**Comment by jimmynotjim** [comment]
If you rebase, you _shouldn't_ need to close this one and open a new one.

**Comment by anselmbradford** [comment]
Just be sure to delete flapjack and re-pull it before rebasing.

**Comment by dpford** [comment]
Closing because this was wrapped into another PR: https://github.com/cfpb/cfgov-refresh/pull/747

**Pull request closed without merging**